app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.kt

```diff
@@ -88,11 +101,14 @@ class SearchActivity : BindingBaseActivity<SearchActivityComponent>() {
         }
     }
 
+    private fun switchTab(soundInfo: SoundInfo) {
+        container.currentItem = if (soundInfo.tabType == TabType.RESULTS) 0 else 1
```

Comment:
There ought to be a better way to couple/define the TabType and its corresponding position.